daemon/daemon_unix.go

```diff
@@ -186,7 +190,10 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
 
 		switch con[0] {
 		case "label":
-			labelOpts = append(labelOpts, con[1])
+			if selinuxConvert {
+				opt = strings.Replace(opt, ":", "=", 1)
```

Comment:
BTW, there's a call to `label.InitLabels` some lines below here. This call needs the whole "label=type:scp_t" but we're not just actually passing "type:scp_t" which leads to having incorrect  process label.